examples/remove_image.yml

```diff
@@ -0,0 +1,12 @@
+---
+#This ansible playbook will remove an image
+
+- hosts: localhost
+  tasks:
+    - name: Test Remove image 
```

Comment:
I would remove the "Test" and just call this "Remove image"

Reply:
Sure, I will make the change